tests/renderer/components/settings-execution-spec.tsx

```diff
@@ -54,47 +57,104 @@ describe('ExecutionSettings component', () => {
     });
   });
 
-  describe('handleExecutionFlagChange()', () => {
-    it('handles new flags', async () => {
-      const wrapper = shallow(<ExecutionSettings appState={store} />);
-      const instance = wrapper.instance() as any;
-      await instance.handleExecutionFlagChange({
-        currentTarget: { value: '--lang=es' },
+  describe('handleItemSettingsChange()', () => {
+    describe('with executionFlags', () => {
```

Comment:
This is more of a personal preference than a strong assertion, but these two tests repeat the same strings a lot e.g. `--lang=es` and `--js-flags=--expose-gc`. One alternative to this repetition would be to make these `const` describe-level variables that the different `it()` tests can use.